articles/logic-apps/error-exception-handling.md

```diff
@@ -332,6 +299,38 @@ To catch exceptions in a **Failed** scope and run actions that handle those erro
 
 For limits on scopes, see [Limits and config](logic-apps-limits-and-config.md).
 
+### Set up a scope with "run after" for exception handling
+
+1. In the [Azure portal](https://portal.azure.com), open your logic app workflow in the designer.
+
+Your workflow must already have a trigger that starts the workflow.
+
+1. On the designer, [follow these generic steps to add a **Control** action named **Scope** to your workflow](/azure/logic-apps/create-workflow-with-trigger-or-action#add-action).
+
+1. In the **Scope** action, [follow these generic steps to the add actions to run](/azure/logic-apps/logic-apps-control-flow-run-steps-group-scopes#add-steps-to-scope), for example:
+
+   ![Screenshot shows workflow designer with actions grouped inside the scope.](./media/error-exception-handling/add-actions-into-scope.png)
```

Comment:
Apologies if I didn't update these links:

```suggestion
   :::image type="content" source="" alt-text="Screenshot shows workflow designer with actions grouped inside the scope.":::
```

Reply:
Should I use the ` :::image` syntax instead of `![]()` because I can not visualize the images with the ` :::image` tag on vscode. Thanks! @ecfan

Reply:
Yes, the ":::image" syntax is the style that the docs now use. The border gets added automatically.